readme.md

```diff
@@ -40,12 +51,32 @@ Clone nivio, build and run a Docker image:
     
   then open http://localhost:8080
   
-https://medium.com/@itzgeoff/including-react-in-your-spring-boot-maven-build-ae3b8f8826e
-   * install nvm oder node > 5
+  ---
+  
+**Docker Setup (Windows)**
  
-https://www.hostingadvice.com/how-to/update-node-js-latest-version/
-
-    nvm install v10.16.3
-    cd src/main/app
-    npm install -g yarn
+  Clone nivio, build and run a Docker image:
+  
+      git clone https://github.com/dedica-team/nivio.git && cd nivio
+      mvn clean package
+      docker build -t nivio:latest .
+      docker run -e SEED=/tmp/inout.yml --mount type=bind,source=C:\<your>\<path>\<to>\nivio\src\test\resources\example,target=/tmp -p 8080:8080 nivio:latest
+      
+   then open http://localhost:8080
+   
+   ---
+ 
+**Development Setup (IntelliJ)**
+ 
+ Create a Spring Boot Configuration in IntelliJ that looks like this:
+ 
+  ![Spring Boot Config](https://raw.githubusercontent.com/dedica-team/nivio/develop/docs/SpringConfig.png)
+  
+  If you need a clean build you can run 
+  
+    mvn clean package
+  
+  If you dont want to use our demo and use your own .yml file instead, replace DEMO=1 with SEED=/path/to/config
```

Comment:
Both SEED and DEMO can be used side by side now.

Reply:
okay, wasn't sure about that part anyway because i didnt used it so far :D i'll change it